vmngclient/api/monitoring_status_api.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING, List
+
+from vmngclient.endpoints.monitoring_status import MonitoringStatus, Status, UpdateStatus
+from vmngclient.typed_list import DataSequence
+
+logger = logging.getLogger(__name__)
+
+if TYPE_CHECKING:
+    from vmngclient.session import vManageSession
+
+
+class MonitoringStatusAPI:
```

Comment:
What's the purpose of this class? The method names and parameters of the method are the same. Return type is almost the same (it just filters out endpoint's return). It would be impossible to maintain it when we build such `API` for every endpoint